lua/ttt2/libraries/database.lua

```diff
@@ -332,7 +332,7 @@ function database.RemoveChangeCallback(accessName, itemName, key, identifier)
 	end
 
 	for i = #callbacks, 1, -1  do
-		callback = callbacks[i]
+		local callback = callbacks[i]
```

Comment:
this is already fixed in master is it not? why does github show this in the diff? :confused:

Reply:
I was confused as well, I even checked the master branch and this makes no sense to me

Reply:
idk why this is showing up this way, but locally the callback is assigned as local